ansible/roles/sidekiq/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+  - name: Install monit
+    apt:
+      name: monit
+      state: present
+    sudo: true
+
+  - name: Install sidekiq dependencies
+    apt:
+      name: "{{item}}"
+      state: present
+    with_items:
+      - redis-server
```

Comment:
there is also a redis role that you should include instead of this.  Maybe the meta.yml here can depend on the redis role?